signer/src/transaction_coordinator.rs

```diff
@@ -827,6 +857,71 @@ where
         }
     }
 
+    fn authenticate_message(
+        packet: &wsts::net::Packet,
+        public_keys: &hashbrown::HashMap<u32, p256k1::point::Point>,
+        public_key_point: p256k1::point::Point,
+        sender_is_coordinator: bool,
+    ) -> bool {
+        macro_rules! check_signer_public_key {
+            ($message:ident) => {
+                match &public_keys.get(&$message.signer_id) {
+                    Some(signer_public_key) if &public_key_point != *signer_public_key => {
+                        tracing::warn!(
+                            ?packet,
+                            reason = "message was signed by the wrong signer",
+                            "ignoring packet"
+                        );
+                        false
+                    }
+                    None => {
+                        tracing::warn!(
+                            ?packet,
```

Comment:
Any thoughts on just having the `packet.msg`, and skip logging the signature (since presumably we have already checked that once we've gotten here).